### PR TITLE
feat: Adds Ticker Metadata_JSON for DYDX

### PR DESCRIPTION
**Pull request by Eric-Warehime**
Adds some structured data which will end up being pushed to x/marketmap in dydx via the `Ticker.Metadata_JSON` field.

### CODE REVIEW

**Review by technicallyty — APPROVED** [review]
LGTM although that is quite a cursed package name lol 😵‍💫

File: x/marketmap/types/tickermetadatajson/dydx.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package tickermetadatajson
```
Comment (aljo242) [inline x/marketmap/types/tickermetadatajson/dydx.go]: can we just make this 

`tickermetatdata`

or 

`metadata`

File: x/marketmap/types/tickermetadatajson/dydx.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	// The price is scaled by Ticker.Decimals.
+	ReferencePrice uint64 `json:"reference_price"`
+	// Liquidity gives a _rough_ estimate of the amount of liquidity in the Providers for a given Market.
+	// It is _not_ updated in coordination with spot prices and only gives rough order of magnitude accuracy at the time
+	// which the update for it is published.
+	// The liquidity value stored here is USD denominated.
+	Liquidity uint64 `json:"liquidity"`
+	// AggregateIds contains a list of AggregatorIDs associated with the ticker.
+	// This field may not be populated if no aggregator currently indexes this Ticker.
+	AggregateIds []AggregatorID `json:"aggregate_ids"`
```
Comment (aljo242) [inline x/marketmap/types/tickermetadatajson/dydx.go]: lint

File: x/marketmap/types/tickermetadata/dydx.go [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	// The price is scaled by Ticker.Decimals.
+	ReferencePrice uint64 `json:"reference_price"`
+	// Liquidity gives a _rough_ estimate of the amount of liquidity in the Providers for a given Market.
+	// It is _not_ updated in coordination with spot prices and only gives rough order of magnitude accuracy at the time
+	// which the update for it is published.
+	// The liquidity value stored here is USD denominated.
+	Liquidity uint64 `json:"liquidity"`
+	// AggregateIDs contains a list of AggregatorIDs associated with the ticker.
+	// This field may not be populated if no aggregator currently indexes this Ticker.
+	AggregateIDs []AggregatorID `json:"aggregate_ids"`
```
Comment (aljo242) [inline x/marketmap/types/tickermetadata/dydx.go:16]:
```suggestion
	BaseAssetID AggregatorID `json:"base_asset_id"`
        QuoteAssetID AggregatorID `json:"quote_asset_id"`
```

I think something like this makes more sense. wee need to track IDs for base and quote asset per CP

**Review by aljo242 — APPROVED** [review]

**Pull request merged**